lighthouse-core/lib/i18n/locales/ar-XB.json

```diff
@@ -1,117 +1,90 @@
 {
   "lighthouse-core/audits/accessibility/accesskeys.js | description": {
-    "message": "‏‮Access‬‏ ‏‮keys‬‏ ‏‮let‬‏ ‏‮users‬‏ ‏‮quickly‬‏ ‏‮focus‬‏ ‏‮a‬‏ ‏‮part‬‏ ‏‮of‬‏ ‏‮the‬‏ ‏‮page‬‏. ‏‮For‬‏ ‏‮proper‬‏ ‏‮navigation‬‏, ‏‮each‬‏ ‏‮access‬‏ ‏‮key‬‏ ‏‮must‬‏ ‏‮be‬‏ ‏‮unique‬‏. [‏‮Learn‬‏ ‏‮more‬‏](https://web.dev/accesskeys/)."
+    "message": "‏‮Access‬‏ ‏‮keys‬‏ ‏‮let‬‏ ‏‮users‬‏ ‏‮quickly‬‏ ‏‮focus‬‏ ‏‮a‬‏ ‏‮part‬‏ ‏‮of‬‏ ‏‮the‬‏ ‏‮page‬‏. ‏‮For‬‏ ‏‮proper‬‏ ‏‮navigation‬‏, ‏‮each‬‏ ‏‮access‬‏ ‏‮key‬‏ ‏‮must‬‏ ‏‮be‬‏ ‏‮unique‬‏. [‏‮Learn‬‏ ‏‮more‬‏](‏‮https‬‏://‏‮dequeuniversity‬‏.‏‮com‬‏/‏‮rules‬‏/‏‮axe‬‏/3.1/‏‮accesskeys‬‏?‏‮application‬‏=‏‮lighthouse‬‏)."
```

Comment:
@exterkamp  ??

Reply:
ok the URL is super old, so maybe it just fell back to the old version of the string, which was  from before us improving the i18n pipeline